labelbox/client.py

```diff
@@ -58,7 +59,8 @@ def __init__(self,
                  endpoint='https://api.labelbox.com/graphql',
                  enable_experimental=False,
                  app_url="https://app.labelbox.com",
-                 rest_endpoint="https://api.labelbox.com/api/v1"):
+                 rest_endpoint="https://api.labelbox.com/api/v1",
+                 adv_rest_endpoint="https://api.labelbox.com/adv"):
```

Comment:
I don't think we want to expose the concept of 'adv' vs 'api' externally, we should only use 1 base gateway api route externally and then just manage different route suffixes per area of functionality

Reply:
The routes for embeddings should probably be something like 
`https://api.labelbox.com/api/v1/data-rows/embedding:POST|PUT|DELETE`, and follow a similar pattern to how we expose other external datarow routes in SDK4 (as something like `https://api.labelbox.com/api/v1/data-rows/`)
cc @kkim-labelbox

Reply:
It should be already there at expressjs, I will check it out later.

Reply:
yep, that sounds right. we should have one proxy route which will then decide where to forward a request

Reply:
I think our offline discussion is that since Embedding is new and doesn't have migration issue as data row, so we can use ADV endpoint directly for now.

Reply:
> The routes for embeddings should probably be something like
> `https://api.labelbox.com/api/v1/data-rows/embeddings:POST|PUT|DELETE`, and follow a similar pattern to how we expose other external datarow routes in SDK4 (as something like `https://api.labelbox.com/api/v1/data-rows/`)

@cssetian that will route the call through the API service, which is what we do not want to do, its an architectural layering violation.  IE, whenver ADV adds an endpoint, that doesn't mean we add a duplicate endpoint in the API service.

Reply:
Note that after talking to chris, we actually have to setup a secondary endpoint system in nginx.